.github/workflows/e2e-nvidia-l40s-x8.yml

```diff
@@ -65,6 +65,11 @@ jobs:
           # https://github.com/actions/checkout/issues/249
           fetch-depth: 0
 
+      - name: Install dependent PRs if needed
+        uses: depends-on/depends-on-action@61cb3f4a0e2c8ae4b90c9448dc57c7ba9ca24c35 # main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Do we want to install dependent PRs for this particular job? This job runs nightly, not in PR builds unless manually triggered. It can't hurt to have this here, but it's just more to maintain if issues were to arise.

@nathan-weinberg  - thoughts?

Reply:
Hmmmmm, we may want this for some manual run scenarios but probably not for the cron-triggered scenarios 

I'm fine to keep it for now